LizarroAnghela/Collections/src/Circles.java

```diff
@@ -0,0 +1,32 @@
+public class Circles {
+    private double radius;
+    private double diameter;
+
+
+
+    public Circles (double radius, double diameter){
+        this.radius = radius;
+        this.diameter = diameter;
+    }
+
+//get set
+    public double getRadius(){
+        return radius;
+    }
+
+    public void setRadius(double radius){
```

Comment:
This class is a little bit tricky because setters are not that quite simple. You can't set radius independently without changing the diameter, as I showed you on the constructor.